### ISSUE TITLE
Dynamic Keys are not extracted correctly

### ISSUE THREAD

**Issue by omardoma** [ngneat/transloco-keys-manager#24]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

Dynamic keys are not extracted correctly

```html
<!-- t('form.errorMessages.404', 'form.errorMessages.default') -->
<article *transloco="let t; read: 'auth.login'">
```

is extracted like this:

```json
  "'form": {
    "errorMessages": {
      "404'": "null",
      "default'": "null"
    }
  }
```

## Expected behavior

Should be extracted correctly and respect the read option in the directive

```json
"login": {
      "form": {
        "errorMessages": {
          "404": "null",
          "default": "null"
        }
}
```

## Environment

<pre><code>
Angular version: 8.2.9

**Comment by shaharkazaz** [comment]
@omardoma This is not a bug since this is currently not supported.
I will add support but the comment will have to be inside the structural and not above it like so: 
```html
<article *transloco="let t; read: 'auth.login'">
<!-- t('form.errorMessages.404', 'form.errorMessages.default') -->
```
Also a question about the JSONs you shared, is auth a scope?

**Comment by omardoma** [comment]
@shaharkazaz Aha, the comment being inside is okay for me and actually is quite intuitive so I believe it would be a great feature. 

Auth is a key in the json file, not a scope.

Also please notice that other than that the read feature is not supported at the moment, there is a bug with the extraction in general, note the single quote in ```"'form"``` as well as the single quote in ```"default'"```

Thank you for the hardwork, transloco has been great!

**Comment by darkv** [comment]
@omardoma When adding translation keys via comments you do not use quotes. That's probably why you see them included in your translation files.

**Comment by omardoma** [comment]
@darkv If that's the case then I guess there is a typo in the README.md that needs to be updated:

<img width="1174" alt="Screen Shot 2020-01-21 at 2 51 39 PM" src="https://user-images.githubusercontent.com/25301162/72806313-9ead3d00-3c5d-11ea-8be1-037585338c67.png">

**Comment by shaharkazaz** [comment]
@omardoma I already fixed the typo, as @darkv said you don't need the quotes.
It's part of some fixes I'm going to release 🙂

**Comment by omardoma** [comment]
@shaharkazaz Awesome, thanks for the hard work, keep it up :)

**Comment by shaharkazaz** [comment]
@omardoma available in version 2.0.0

**Comment by omardoma** [comment]
@shaharkazaz Thanks a lot, works great!
